integration_tests/src/main/python/json_test.py

```diff
@@ -417,6 +417,16 @@ def test_from_json_struct_of_list(data_gen, schema):
             .select(f.from_json(f.col('a'), schema)),
         conf={"spark.rapids.sql.expression.JsonToStructs": True})
 
+@pytest.mark.parametrize('data_gen', [StringGen('', nullable=True)])
```

Comment:
If there is only one value for `data_gen`, we don't need to parametrize it. Instead just create the `StringGen`  directly inside the test function.

Reply:
I believe it's two but hard to see